app/collections/preview.cjsx

```diff
@@ -43,13 +45,15 @@ module.exports = React.createClass
         </p>
         <div className="subject-previews">
           {if @state.subjects
-            <div>
-              {for subject in @state.subjects
-                <Thumbnail
-                  key={"collection-preview-#{@props.collection.id}-#{subject.id}"}
-                  src={getSubjectLocation(subject).src}
-                  width={100} />}
-            </div>
+            <Link to="/projects/#{@props.project.slug}/collections/#{@props.collection.slug}">
+              <div>
+                {for subject in @state.subjects
+                  <Thumbnail
+                    key={"collection-preview-#{@props.collection.id}-#{subject.id}"}
+                    src={getSubjectLocation(subject).src}
+                    width={radius} />}
```

Comment:
This prop just sets the `max-width` style on the image in the thumbnail, so `radius` isn't a really clear variable name to me.